composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerScreen.kt

```diff
@@ -45,7 +45,9 @@ import org.jetbrains.compose.ui.tooling.preview.Preview
 @Preview(showBackground = true)
 @Composable
 fun TimerScreenPreview() {
-    val viewModel = TimerViewModel()
+    val viewModel = TimerViewModel(
+        router = {},
+    )
```

Comment:
The `router` in the Preview isn't doing anything, so when you click the button in the Preview nothing will happen.